nengo/cache.py

```diff
@@ -45,6 +45,14 @@ def safe_remove(path):
         logger.warning("OSError during safe_remove: %s", err)
 
 
+def safe_makedirs(path):
+    if not os.path.exists(path):
+        try:
+            os.makedirs(path)
+        except OSError as err:
```

Comment:
I was thinking about that. I did not do it to keep it equivalent to the other `safe_<foo>` functions. If we do that it probably would be better to check the error code instead of the string, but Windows might give a weird error number that is not in `errno` in that case I believe.